test/index.js

```diff
@@ -85,6 +89,15 @@ const server = http.createServer((request, response) => {
   });
 })();
 
+async function downloadLatestAmpMeta() {
+  const METADATA_URL = 'https://cdn.ampproject.org/rtv/metadata';
+  const ampRuntimeMeta = await (await fetch(METADATA_URL)).json();
+  await writeFile(
+    __dirname + '/amp_metadata.json',
```

Comment:
Recommend using `path` here to merge paths.

Reply:
done